agents/trials.js

```diff
@@ -57,8 +57,13 @@ function searchTrials(query, page, perPage, filters) {
     .then((response) => response.obj);
 }
 
+function searchTrialsByEntity(entityType, entityName) {
+  return searchTrials(`${entityType}:${entityName}`, 1, 10);
```

Comment:
@vitorbaptista: I think, we don't should escape `entityName` at this side. I have tested request `person:hello " world` and i got escaped value on API side `person:hello \" world` (and yes, i got some break)
In my opinion, we should process query string on API side only.